### PR TITLE
Add type definitions for Yandex.Games SDK

### PR DESCRIPTION
**Pull request by koteelok**
# Description

These are typings that would be very useful when working with Yandex.Games SDK. There is no npm package for this SDK, it's only meant for browser games.

You can find more information here: https://yandex.ru/dev/games/doc/en/

# Template

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:

- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

If removing a declaration:

- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/ysdk/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+    | "tl"
+    | "tr"
+    | "uk"
+    | "ur"
+    | "uz"
+    | "vi"
+    | "zh"
+    | "zu";
+
+/* prettier-ignore */
```
Comment (peterblazejewicz) [inline types/ysdk/index.d.ts:335]: please remove that one and other one, DT is now using `dtprint` for formatting (refers to readme for details)

**Comment by peterblazejewicz** [comment]
Please add tests to cover core features, like on dev documentation section. Types looks ok, but let's prove it.

**Comment by koteelok** [comment]
I can't add tests because it's the browser-only API that works specifically on the Yandex.Games site.

That's why there is no npm package for these types.

We are constantly working with our developer community to make sure that the typings are up to date.

**Comment by koteelok** [comment]
@peterblazejewicz 

I can't add tests because it's the browser-only API that works specifically on the Yandex.Games site.

That's why there is no npm package for these types.

We are constantly working with our developer community to make sure that the typings are up to date.

**Comment by koteelok** [comment]
@peterblazejewicz

This is exactly what I'm trying to say. That project **CAN'T** be covered with tests because it's typings for [Yandex.Games SDK](https://yandex.ru/dev/games/doc/en/sdk/sdk-about). That SDK is avaliable **ONLY** for [browser games](https://yandex.ru/games/).

**Comment by peterblazejewicz** [comment]
Let me rephrase. `jQuery` is run within browser, but types for jQuery are here and there are tests for jQuery:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jquery/jquery-tests.ts
Please add a test written in same way as users of SDK would use them in browsers, thank you!

**Comment by koteelok** [comment]
@peterblazejewicz

Thanks for the guidance, I looked at the example you provided and implemented tests for this package in a similar way.

Can you take a look and let me know if there are any more issues?

File: types/ysdk/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "projects": [
+        "https://yandex.ru/dev/games/doc/en/"
+    ],
+    "devDependencies": {
+        "@types/ysdk": "workspace:."
+    },
+    "owners": [
+        {
+            "name": "koteelok",
+            "githubUsername": "yandex-games-plugins"
```
Comment (peterblazejewicz) [inline types/ysdk/package.json:16]:
```suggestion
            "name": "Musa Asukhanov",
            "githubUsername": "koteelok"
```
an organisation cannot be owner (requires user). From Readme:
> DT has the concept of "Definition Owners" which are people who want to maintain the quality of a particular module's types.
Merging this to avoid nitpicking, you can take this by deciding if it's ok to merge

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@koteelok thanks!

**Comment by koteelok** [comment]
Ready to merge
